applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -292,6 +292,12 @@ void le_audio_conn_set(struct bt_conn *conn)
 	LOG_WRN("No conn used in BIS");
 }
 
+int le_audio_pa_sync_set(struct bt_le_per_adv_sync *pa_sync, uint32_t broadcast_id)
+{
+	LOG_WRN("Not used in BIS gateway");
```

Comment:
Add ARG_UNUSED

Reply:
Done, I went through all the le_audio_files and added ARG_UNUSED there as well, for consistency